### PR TITLE
ksphere/kommander: Fix konvoy 1.5 docs URLs

### PR DESCRIPTION
**Pull request by dlipovetsky**
## Jira Ticket
https://jira.d2iq.com/browse/D2IQ-69908

## Description of changes being made
Fix konvoy 1.5 docs URLs. Refer to 1.5 instead of 1.5.0-beta. Merge after konvoy 1.5 is released. 

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Provide an estimated date for deploying the doc change. Note: Improvements or fixes can be merged ASAP. 
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/1.1.0-beta/clusters/attach-eks-cluster/index.md [hunk 1]
```diff
@@ -4,18 +4,18 @@ navigationTitle: Attach Amazon EKS Cluster to Kommander
...
+You can attach existing Kubernetes clusters to Kommander. After attaching the cluster, you can use Kommander to [examine and manage](/ksphere/kommander/1.1.0-beta/clusters/) this cluster. The following procedure shows how to attach an existing Amazon Elastic Kubernetes Service (EKS) cluster to Kommander.
 
-## Before you begin 
+## Before you begin
 
 This procedure requires the following items and configurations:
 
 - A fully configured and running Amazon [EKS](https://aws.amazon.com/eks/) cluster with administrative privileges.
-- Konvoy v1.5.0-beta.0 or above, [installed and configured](/ksphere/konvoy/1.5.0-beta/install/) for your Amazon EKS cluster, on your machine.
+- Konvoy v1.5.0-beta.0 or above, [installed and configured](/ksphere/konvoy/1.5/install/) for your Amazon EKS cluster, on your machine.
```
Comment (msdolbey) [inline pages/ksphere/kommander/1.1.0-beta/clusters/attach-eks-cluster/index.md:18]: Should we change this to just `Konvoy v1.5` ?
```suggestion
- Konvoy v1.5 or above, [installed and configured](/ksphere/konvoy/1.5/install/) for your Amazon EKS cluster, on your machine.
```

File: pages/ksphere/kommander/1.1.0-beta/clusters/attach-eks-cluster/index.md [hunk 2]
```diff
@@ -132,12 +132,12 @@ This procedure requires the following items and configurations:
 
 For information on related topics or procedures, refer to the following:
 
-- [Configuring and Running Amazon EKS Clusters](https://aws.amazon.com/eks/) 
+- [Configuring and Running Amazon EKS Clusters](https://aws.amazon.com/eks/)
 
-- [Installing and Configuring Konvoy v1.5.0-beta.0 or above](/ksphere/konvoy/1.5.0-beta/install/)
+- [Installing and Configuring Konvoy v1.5.0-beta.0 or above](/ksphere/konvoy/1.5/install/)
```
Comment (msdolbey) [inline pages/ksphere/kommander/1.1.0-beta/clusters/attach-eks-cluster/index.md:137]: Same here, should it be just v1.5 or nah?
```suggestion
- [Installing and Configuring Konvoy v1.5 or above](/ksphere/konvoy/1.5/install/)
```

File: pages/ksphere/kommander/1.1.0-beta/clusters/attach-eks-cluster/index.md (hunk 1 above)
Comment (dlipovetsky) [reply pages/ksphere/kommander/1.1.0-beta/clusters/attach-eks-cluster/index.md:18]: Good catch, thank you. Will fix.

File: pages/ksphere/kommander/1.1.0-beta/clusters/attach-eks-cluster/index.md (hunk 2 above)
Comment (dlipovetsky) [reply pages/ksphere/kommander/1.1.0-beta/clusters/attach-eks-cluster/index.md:137]: Good catch, thank you. Will fix.

**Comment by dlipovetsky** [comment]
Putting this on hold until Konvoy 1.5 is released.

**Comment by msdolbey** [comment]
Actually, on second thought @dlipovetsky - we shouldn't block this, the link structure has already changed on the docs site publically, so I think we should merge this. FYI @cmastr

**Comment by cmastr** [comment]
> Actually, on second thought @dlipovetsky - we shouldn't block this, the link structure has already changed on the docs site publically, so I think we should merge this. FYI @cmastr

Yeah, I agree. We don't need to hold these until Konvoy goes out. The docs are already live.

**Pull request merged**